src/Components/Cart/utilities/useOrderSummary.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { useStateProvider } from "../../../Context/StateContext/state-provider";
+import { cartPriceCalculator } from "./cart-price-calculator";
+
+export const useOrderSummary = () => {
+  const { userState } = useStateProvider();
+  const cartDetails = cartPriceCalculator(userState.cartItems);
+  const deliveryCharges =
+    cartDetails.totalPrice - cartDetails.discount > 10000 ? 0 : 40;
```

Comment:
- You can make this value `10,000` as a const and use it there instead of writing it manually. 
- ex. 
`Const DELEIVERY_AMOUNT_SELECTOR = 10,000;`
`const DELEIVERY_CHARGES=40;`
`cartDetails.totalPrice - cartDetails.discount > DELEIVERY_AMOUNT_SELECTOR ? 0 : DELEIVERY_CHARGES;`